naas/requests/project_subscriber_profiles.py

```diff
@@ -0,0 +1,67 @@
+import json
+from naas.client import Client
+
+
+class ProjectSubscriberProfiles:
+
+    @staticmethod
+    def retrieve_by_project_id_and_project_subscriber_id(
+            project_id, project_subscriber_id, params=None):
+        """
+        Retrieve the instance of a project subscriber by project
+
+        :param project_id: int
+        :param project_subscriber_id: int
+        :param params: dict
+        :return: Response
+        """
+        if params is None:
+            params = {}
+
+        rel = Client.rel_for('rels/project-property')
```

Comment:
@arthuroe - this needs to be `rels/project-subscriber-profile`. This is returning the wrong data set.